kazuki-core-test/src/test/kotlin/com/anaplan/engineering/kazuki/core/test/TestPrimitiveInvariant.kt

```diff
@@ -0,0 +1,129 @@
+package com.anaplan.engineering.kazuki.core.test
```

Comment:
Thanks for this very thorough and just what we're looking for..makes it so much easier to implement when we have a bank of failing requirements to satisfy :+1: